core/trino-main/src/main/java/io/trino/execution/scheduler/NodeSchedulerConfig.java

```diff
@@ -108,8 +109,20 @@ public NodeSchedulerConfig setIncludeCoordinator(boolean includeCoordinator)
         return this;
     }
 
-    @Config("node-scheduler.max-pending-splits-per-task")
+    @Config("node-scheduler.min-pending-splits-per-task")
```

Comment:
I think we should instead do:
```
@Config("node-scheduler.min-pending-splits-per-task")
@LegacyConfig({"node-scheduler.max-pending-splits-per-task", "node-scheduler.max-pending-splits-per-node-per-task", "node-scheduler.max-pending-splits-per-node-per-stage"}
```
and have:
```
"node-scheduler.max-adjusted-pending-splits-per-task"
```
It's still a breaking change, but at least it won't confuse everyone with `min/max` naming that that `max` only relates to uniform scheduler